specs/linux/bpf_file_events.table

```diff
@@ -0,0 +1,22 @@
+table_name("bpf_file_events")
```

Comment:
As a slight nit, we're trying to move this to putting the variant at the end -- `file_events_bpf`